packages/preview/minimalbc/0.0.1/lib.typ

```diff
@@ -0,0 +1,88 @@
+#let companytext(size, body, style: "normal") = text(
+  size: size,
+  style: style,
+  weight: "medium",
+)[#body]
+#let nametext(size, body, style: "normal") = text(
+  size: size,
+  style: style,
+  weight: "medium",
+)[#body]
+#let roletext(size, body, style: "normal") = text(
+  size: size,
+  style: style,
+  weight: "light",
+)[#body]
+
+#let minimalbc(
+  company_name: "Company Name",
```

Comment:
Typst packages generally use kebab-case names (but this is only a convention and you can keep snake_case if you prefer).